bottom/client.py

```diff
@@ -102,8 +102,9 @@ def trigger(self, event: str, **kwargs: Any) -> None:
         async_event.set()
         async_event.clear()
 
-    async def wait(self, event: str) -> None:
+    async def wait(self, event: str) -> str:
         await self._events[event.upper()].wait()
+        return event
```

Comment:
Should this return the normalized value `event.upper()` that we're keying off of, or the original value `event` that the caller may use as a key?

I think you're correct with just `event` but whichever we pick more or less can't change in the future.  This would be a bad breaking change even in a major version (no discoverability, silent failure)